text/0111-feature-flags.md

```diff
@@ -154,9 +154,30 @@ describe('foo', () => {
 });
 ```
 
+Note that this will only work for features behind flags that are not used during the initialization of the engine.
+
+```js
+// Works when run as part of initialization
+const foo = () => {
+    if (ENABLE_FOO) {
+        return 1;
+    } else {
+        return 2;
+    }
+};
+
+// Doesn't work when run as part of initialization
+let foo;
+if (ENABLE_FOO) {
+    foo = () => 1;
+} else {
+    foo = () => 2;
+}
+```
+
 ### Extensibility
 
-It is possible that `@lwc/config` can be used beyond this package, so other parts of the platform can implement a similar solution for their own features. This might include components, which means that the LWC compiler will have to understand the transformation as well.
+It is possible that `@lwc/features` can be used beyond this package, so other parts of the platform can implement a similar solution for their own features. This might include components, which means that the LWC compiler will have to understand the transformation as well.
```

Comment:
is this a warning? Who's responsible in enduring that other entities understand the import?

Reply:
This is just saying that @lwc/features could be be used outside of our repository so we should keep code reuse in mind.